quota/etcd/README.md

```diff
@@ -0,0 +1,246 @@
+# Etcd quotas
+
+Package etcd (and its subpackages) contain an etcd-based
+[quota.Manager](https://github.com/google/trillian/blob/3cf59cdfd0/quota/quota.go#L101)
+implementation, with a corresponding REST-based configuration service.
+
+## Usage
+
+First, ensure both logserver and logsigner are started with the `--etcd_servers`
+and `--quota_system=etcd` flags, in addition to other flags. Logserver must
+also be started with a non-empty `--http_endpoint` flag, so the REST quota API
+is bound.
+
+For example:
+
+```bash
+trillian_log_server \
+  --etcd_servers=... \
+  --http_endpoint=localhost:8091 \
+  --quota_system=etcd
+
+trillian_log_signer --etcd_servers=... --quota_system=etcd
+```
+
+If correctly started, the servers will be using etcd quotas. The default
+configuration is empty, which means no quotas are enforced.
+
+The REST quota API may be used to create and update configurations.
+
+For example, the command below creates a sequencing-based, global/write quota.
+Assuming an expected sequencing performance of 50 QPS, the max_tokens specified
+implies a backlog of 4h.
+
+```bash
+curl \
+  -d '@-' \
+  -s \
+  -H 'Content-Type: application/json' \
+  -X POST \
+  "localhost:8091/v1beta1/quotas/global/write/config" <<EOF
+{
+  "name": "quotas/global/write/config",
+  "config": {
+    "state": "ENABLED",
+    "max_tokens": 288000,
+    "sequencing_based": {
+    }
+  }
+}
+EOF
+```
+
+To list all configured quotas, do:
+
+```bash
+curl localhost:8091/v1beta1/quotas?view=FULL
```

Comment:
probably better single quote escape that URL

Reply:
Good call, done.